src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write code here
+const tableHead = document.querySelector('table thead');
+const tableBody = document.querySelector('table tbody');
+const tableRows = [...document.querySelectorAll('table tbody tr')];
+
+tableHead.addEventListener('click', e => {
+  const criterion = [...tableHead.children[0].children].indexOf(e.target);
+
+  tableRows.sort((a, b) => {
+    const textContentA = a.children[criterion].textContent;
+    const textContentB = b.children[criterion].textContent;
+
+    if (criterion === 2) { // sorting by age
```

Comment:
switch case

Reply:
also, it's not the best idea to sort table depending on cellIndex, cause they may change, so it's better to come up with more universal solution